scripts/docker/build.gradle

```diff
@@ -0,0 +1,174 @@
+//Note: this section 'buildscript` is only for the dependencies of the buildscript itself.
+// See the second 'repositories' section below for the actual dependencies of GATK itself
+buildscript {
+    repositories {
+        mavenCentral()
+        jcenter() // for shadow plugin
+     }
+}
+
+
+plugins {
+    id "java"           // set up default java compile and test tasks
+    id 'maven'
+    id "jacoco"
+}
+
+repositories {
+    mavenCentral()
+    jcenter()
+    maven {
+        url "https://broadinstitute.jfrog.io/broadinstitute/libs-snapshot/" //for htsjdk snapshots
+    }
+    mavenLocal()
+}
+
+////Note: the test suite must use the same defaults. If you change system properties in this list you must also update the one in the test task
+//applicationDefaultJvmArgs = ["-Dsamjdk.use_async_io_read_samtools=false","-Dsamjdk.use_async_io_write_samtools=true", "-Dsamjdk.use_async_io_write_tribble=false", "-Dsamjdk.compression_level=2"]
+
+tasks.withType(Test) {
+    outputs.upToDateWhen { false }  //tests will never be "up to date" so you can always rerun them
+    String TEST_VERBOSITY = "$System.env.TEST_VERBOSITY"
+
+    /**
+     * Valid options for TEST_TYPE are:
+     * cloud, integration, unit  : run one of the three disjoint partitions of the test suite
+     * all                       : run all the tests
+     * anything else             : run the non-cloud tests
+     */
+    String TEST_TYPE = "$System.env.TEST_TYPE"
+
+    useTestNG {
```

Comment:
We need to add a big obvious comment to this identical block in the regular build.gradle that this has to be updated when it is.  Alternatively, can we factor it out into a shared subscript?  We're going to have nasty bugs where we aren't running some tests when someone updates the regular test block but not this one.

Reply:
If we can share the code between the two scripts, can we merge this tasks.withType block with the actual test below, it's weird to split it when there's only 1.